src/test/java/bio/terra/service/resourcemanagement/azure/AzureResourceConfigurationTest.java

```diff
@@ -288,17 +288,29 @@ public void testAbilityToDeployApplication() {
                   .credential(new AzureNamedKeyCredential(storageAccountName, key))
                   .endpoint("https://" + storageAccountName + ".table.core.windows.net")
                   .buildClient();
-          TableClient tableClient = tableServiceClient.createTable("files");
-          String datasetId = UUID.randomUUID().toString();
+
+          // Add an entry
```

Comment:
I saw another connected test use an existing storage account, so I'll probably move this into it's own test

Reply:
Right, I think that makes sense